test/functorch/test_control_flow.py

```diff
@@ -256,6 +322,15 @@ def setUp(self):
         torch._dynamo.reset()
         super().setUp()
 
+    def _check_tracing(self, fn, args):
+        graphs = []
+        eager_res = fn(*args)
+        for tracing_mode in ["symbolic", "real", "fake"]:
+            graph = make_fx(fn, tracing_mode=tracing_mode)(*args)
+            graphs.append(graph)
```

Comment:
nit: graphs as a dict or namedtuple would make the code more readable; e.g. graphs["symbolic"] would get the symbolic graph.